src/main/java/weiliang/bot/task/Task.java

```diff
@@ -0,0 +1,29 @@
+package weiliang.bot.task;
+
+public class Task {
```

Comment:
I think adding header comments in the different files in your project would be helpful to aid readers to understand their main purpose and functionality :D